server/routes/users.js

```diff
@@ -1,5 +1,6 @@
 import { Router } from 'express';
-import UsersController from '../controllers/UsersController';
+import UsersController from '../controllers/UserController';
```

Comment:
You also need to change your imports accordingly after renaming the class, to avoid pointing to an unexisting object.